src/components/common/dashboard/Navbar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Link, Button, Flex, Heading, Icon, Image, VStack } from '@chakra-ui/react';
+import { FiUsers } from 'react-icons/fi';
+import { IoHomeOutline, IoPodiumOutline } from 'react-icons/io5';
+import { Link as RouterLink, NavLink } from 'react-router-dom';
+import Logo from '../../../assets/logoWhite.png';
+
+const NavItem = props => (
+  <Link as={NavLink} to={props.to} _activeLink={{ color: 'white' }}>
+    <Heading size="h4" as="span">
+      <Flex>
+        <Icon as={props.icon} mr={2} />
+        {props.children}
```

Comment:
Kind of unrelated comment as I've noticed this on other branches as well but I've noticed that when you click on the nav item (or just anything you need to click like buttons), there's this blue outline that doesn't go away until we click something else. Is this expected?
<img width="215" alt="blue outline" src="https://user-images.githubusercontent.com/28817310/109432533-a51df480-79d9-11eb-94a4-d6ae189d118b.png">

Reply:
The outline is there for a11y reasons. Checkout http://www.outlinenone.com/